homeassistant/components/webostv/manifest.json

```diff
@@ -6,5 +6,6 @@
   "requirements": ["aiowebostv==0.1.1", "sqlalchemy==1.4.27"],
   "codeowners": ["@bendavid", "@thecode"],
   "ssdp": [{"st": "urn:lge-com:service:webos-second-screen:1"}],
-  "iot_class": "local_polling"
+  "quality_scale": "platinum",
```

Comment:
We require full test coverage, above average per module, for gold level.

Reply:
Ok, I will work on getting them to 100% 👍

Reply:
With the following PRs the integration has 100% coverage for all modules: 
- https://github.com/home-assistant/core/pull/64723
- https://github.com/home-assistant/core/pull/64801